### PR TITLE
Reposition Editor Reopen With action.

### LINKED ISSUE #101463: Show editor reopen with in title context menu
For mouse users, the title context menu might be a better place for the reopen action

image

cc @mjbvz

### PR DESCRIPTION
**Pull request by rebornix**
This PR fixes https://github.com/microsoft/vscode/issues/101463. Based on the discussion we had this morning, I created a new group `1_editorviewtype`.

image

It's not next to `1_close` group as *Copy Path* actions are in group `1_cutcopypaste` and it's hard to find a meaningful group name between `1_cl...` and `1_cu..` (suggestions welcome)

### CODE REVIEW

**Review by mjbvz — APPROVED** [review]

**Comment by mjbvz** [comment]
I like this ordering. We can also move the markdown preview command to use the new group

File: src/vs/workbench/browser/parts/editor/editor.contribution.ts [hunk 1]
```diff
@@ -446,6 +446,9 @@ MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCo
 MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCommands.CLOSE_EDITORS_TO_THE_RIGHT_COMMAND_ID, title: nls.localize('closeRight', "Close to the Right"), precondition: EditorGroupEditorsCountContext.notEqualsTo('1') }, group: '1_close', order: 30, when: ContextKeyExpr.has('config.workbench.editor.showTabs') });
 MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCommands.CLOSE_SAVED_EDITORS_COMMAND_ID, title: nls.localize('closeAllSaved', "Close Saved") }, group: '1_close', order: 40 });
 MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCommands.CLOSE_EDITORS_IN_GROUP_COMMAND_ID, title: nls.localize('closeAll', "Close All") }, group: '1_close', order: 50 });
+
+MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: ReopenResourcesAction.ID, title: ReopenResourcesAction.LABEL }, group: '1_editorviewtype', order: 10, when: ActiveEditorAvailableEditorIdsContext });
+
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/editor/editor.contribution.ts:451]:
```suggestion
MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: ReopenResourcesAction.ID, title: ReopenResourcesAction.LABEL }, group: '1_open', order: 10, when: ActiveEditorAvailableEditorIdsContext });
```

Comment (bpasero) [reply src/vs/workbench/browser/parts/editor/editor.contribution.ts:451]: Isn't maybe `1_open` a better name? Then this PR should also include to move "Open Preview" into there imho @mjbvz

**Review by bpasero — CHANGES_REQUESTED** [review]

File: src/vs/workbench/browser/parts/editor/editor.contribution.ts (hunk 1 above)
Comment (rebornix) [reply src/vs/workbench/browser/parts/editor/editor.contribution.ts:451]: I like `1_open`. It looks great after moving "Open Preview" into it

image

@mjbvz

File: src/vs/workbench/browser/parts/editor/editor.contribution.ts [hunk 2]
```diff
@@ -446,6 +446,9 @@ MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCo
 MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCommands.CLOSE_EDITORS_TO_THE_RIGHT_COMMAND_ID, title: nls.localize('closeRight', "Close to the Right"), precondition: EditorGroupEditorsCountContext.notEqualsTo('1') }, group: '1_close', order: 30, when: ContextKeyExpr.has('config.workbench.editor.showTabs') });
 MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCommands.CLOSE_SAVED_EDITORS_COMMAND_ID, title: nls.localize('closeAllSaved', "Close Saved") }, group: '1_close', order: 40 });
 MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: editorCommands.CLOSE_EDITORS_IN_GROUP_COMMAND_ID, title: nls.localize('closeAll', "Close All") }, group: '1_close', order: 50 });
+
+MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: ReopenResourcesAction.ID, title: ReopenResourcesAction.LABEL }, group: '1_open', order: 10, when: ActiveEditorAvailableEditorIdsContext });
+
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/editor/editor.contribution.ts:451]:
```suggestion
MenuRegistry.appendMenuItem(MenuId.EditorTitleContext, { command: { id: ReopenResourcesAction.ID, title: ReopenResourcesAction.LABEL }, group: '1_open', order: 10, when: ActiveEditorAvailableEditorIdsContext });
```

**Review by bpasero — CHANGES_REQUESTED** [review]

**Pull request merged**